docs/workspaces/index.md

```diff
@@ -64,6 +64,7 @@ The question's `info.json` should set the `singleVariant` and `workspaceOptions`
     * `args` (optional, default none): command line arguments to pass to the Docker image
     * `syncIgnore` (optional, default none): list of files or directories that will be excluded from sync
     * `rewriteUrl` (optional, default true): if true, the URL will be rewritten such that the workspace container will see all requests as originating from /
+    * `enableNetworking` (optional, default false): whether the workspace should be allowed to connect to the public internet. This is disabled by default to make secure, isolated execution the default behavior. This restriction is not enforced when running PrairieLearn in local development mode.
```

Comment:
What's the likelihood that this will break existing workspaces? I think this'll be worth it regardless, but might warrant advanced warning to folks before deploying.

Reply:
Good point, I should have documented the deployment plan. My intention is to deploy in a non-enforcing mode, so that people have a chance to add `"enableNetworking":true` to their `info.json` files. At this point I'll send email to anyone using workspaces and warn them about the change, with a deadline (2 weeks?) for when we'll start enforcing it.

An alternative would be to wait until after final exams are over at UIUC and then just start enforcing it. A news item about the change might be a good idea in this case?